src/midi_device.rs

```diff
@@ -1,17 +1,10 @@
-use crate::data::usb::constants::*;
-use crate::data::usb_midi::usb_midi_event_packet::{MidiPacketParsingError, UsbMidiEventPacket};
-use usb_device::class_prelude::*;
-use usb_device::Result;
+use crate::{constants::*, event_packet::UsbMidiEventPacket};
+use usb_device::{class_prelude::*, Result};
 
 const MIDI_IN_SIZE: u8 = 0x06;
 const MIDI_OUT_SIZE: u8 = 0x09;
```

Comment:
Should these be moved to constants?

Reply:
they're not `pub`, only used in this file, so I don't think so